.github/workflows/maintainer_bors.yml

```diff
@@ -0,0 +1,93 @@
+name: Add "ready-to-merge" and "delegated" label
+
+# triggers the action when
+on:
+  # the PR receives a comment
+  issue_comment:
+    types: [created]
+  # the PR receives a review
+  pull_request_review:
+    # whether or not it is accompanied by a comment
+    types: [submitted]
+  # the PR receives a review comment
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  add_ready_to_merge_label:
+    # we set some variables. The ones of the form `${{ X }}${{ Y }}` are typically not
+    # both set simultaneously: depending on the event that triggers the PR, usually only one is set
```

Comment:
"typically" and "usually" -- what happens in the edge where both are set? And do we know what kind of edge cases exist?

Reply:
Yes, this is all in the GH API and it should be as follows.
* Variables that contain `comment` are defined when the action is... a comment!
* Variables that contain `review` could be defined on both `review` or `review_comment`.
* `event_name` will distinguish between `comment`, `review` and `review_comment`.